docs/content/src/learn/guides/database-integrations/querying-mysql-database/querying-mysql-database/src/query_employee_data/main.bal

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/config;
+import ballerina/http;
+import ballerina/jsonutils;
+import ballerina/log;
+import ballerinax/java.jdbc;
+
+jdbc:Client employeeDB = new({
+    url: config:getAsString("MYSQL_URL"),
+    username: config:getAsString("MYSQL_USERNAME"),
+    password: config:getAsString("MYSQL_PASSWORD"),
+    poolOptions: { maximumPoolSize: 5 },
```

Comment:
IMO, It is better not to specify poolOptions in the guides as that will create a seperate pool for this client. Instead, we can reuse the global pool if we omit this. So unless it is a must we shouldn't encourage changing the pool configs. Refer https://preprod.ballerina.io/learn/api-docs/ballerina/java.jdbc/index.html for more details about "Connection Pool Handling"

Reply:
Fixed